resources/components/ImageCard.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+	<div class="wbmad-image-with-suggestions-container">
+		<div class="wbmad-image-with-suggestions-image">
+			<div class="wbmad-image-with-suggestions-image-wrapper">
+				<a v-bind:href="descriptionUrl" target="_blank">
+					<img v-bind:src="thumbUrl" alt="">
+				</a>
+			</div>
+		</div>
+
+		<div class="wbmad-image-with-suggestions-tags">
+			<div class="wbmad-suggestion-group">
+				<ul>
+					<li v-for="(suggestion, index) in suggestions"
+						v-bind:key="index"
+						v-on:click="toggleConfirmed( suggestion )">
+						{{ suggestion }}
```

Comment:
The next step here will be to build a <suggestion> component that takes the suggestion data as props and can change how it is displayed based on the confirmed state.